charts/br-sta/values.yaml

```
@@ -0,0 +1,585 @@
+  #   configmap.<NATIVE_KEY> (here) > br-sta.<group>.<field> > chart default.
+  # Defaults live in the template, so leaving this {} renders the documented
+  # defaults. Set a raw key here only to override a value or pass an opt-in key.
+  # @default -- templates/configmap.yaml
+  configmap: {}
+  # -- Grouped config API (see templates/configmap.yaml for field names). Postgres/
+  # Redis/RabbitMQ HOST/PORT/USER come from the datastores mask; MT/OTEL from the
+  # global.* blocks. These groups hold the remaining per-service knobs.
+  app: {}          # ENV_NAME(envName) / LOG_LEVEL(logLevel) / DEPLOYMENT_MODE / etc.
+  server: {}       # SERVER_ADDRESS / HTTP_BODY_LIMIT_BYTES / TLS_TERMINATED_UPSTREAM ...
+  cors: {}         # allowedOrigins / allowedMethods / allowedHeaders / exposeHeaders / allowCredentials
+  multiTenant: {}  # defaultTenantId (enabled lives in br-sta.multiTenant.enabled)
```

YAMLlint (1.37.1)

[error] duplication of key "multiTenant" in mapping

(key-duplicates)